examples/EvaluatingChatAssistant/EvaluatingChatAssistant.ipynb

```diff
@@ -27,306 +29,691 @@
+    "        })\n",
+    "    );\n",
+    "\n",
+    "    const response = await client.chat.completions.create({\n",
+    "        model: \"gpt-4o\",\n",
+    "        messages: [\n",
+    "            {\n",
+    "                role: \"system\",\n",
+    "                content: \"You are a helpful and polite assistant who knows about sports.\"\n",
+    "            }, {\n",
+    "                role: \"user\",\n",
+    "                content: input\n",
+    "            }\n",
+    "        ]\n",
+    "    });\n",
+    "    return response.choices[0].message.content || \"\";\n",
+    "}"
    ]
   },
   {
    "cell_type": "markdown",
+   "id": "53783a49-1c41-4668-92e3-142887534c00",
    "metadata": {},
    "source": [
-    "## Running evals\n",
+    "#### Scoring and running the eval\n",
     "\n",
-    "The key to running evals on a multi-turn conversation is to include the history of the chat in the chat completion request.\n",
+    "We'll use the `Factuality` scoring function from the [autoevals library](https://www.braintrust.dev/docs/reference/autoevals) to check how the output of the chat completion compares factually to the expected value and specify for the scorer to use `gpt-4o` ([docs for using scorers with additional fields](https://www.braintrust.dev/docs/guides/evals/write#scorers-with-additional-fields)).\n",
     "\n",
-    "To do so, we simply add the history between the `system` message and the final `user` message in the `messages` argument as seen below."
+    "We will also utilize [trials](https://www.braintrust.dev/docs/guides/evals/write#trials) by including the `trialCount` parameter in the `Eval` call. We expect the output of the chat completion to be non-deterministic, so running each input multiple times will give us a better sense of the \"average\" output."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
+   "id": "a6842138-4cf1-4210-8dfa-777a8833c9d0",
    "metadata": {},
    "outputs": [],
    "source": [
-    "import { wrapOpenAI } from \"braintrust\";\n",
-    "import { OpenAI } from \"openai\";\n",
+    "import { Eval } from \"braintrust\";\n",
+    "import Factuality, { Score } from \"autoevals\";\n",
     "\n",
-    "async function runTask({\n",
-    "  input,\n",
-    "  chat_history\n",
-    "}: {\n",
-    "  input: string,\n",
-    "  chat_history: {\n",
-    "    role: \"user\" | \"assistant\";\n",
-    "    content: string;\n",
-    "  }[]\n",
-    "}) {\n",
-    "  const client = wrapOpenAI(\n",
-    "    new OpenAI({\n",
-    "      baseURL: \"https://braintrustproxy.com/v1\",\n",
-    "      apiKey: process.env.OPENAI_API_KEY ?? \"\", // Can use OpenAI, Anthropic, Mistral etc. API keys here\n",
-    "    })\n",
-    "  )\n",
+    "Eval(\"Chat assistant\", {\n",
+    "    experimentName: \"gpt-4o assistant - no history\",\n",
+    "    data: () => experimentData,\n",
+    "    task: runTask,\n",
+    "    scores: [Factuality.partial({ model: \"gpt-4o\" })],\n",
```

Comment:
We should land https://github.com/braintrustdata/autoevals/pull/74 and update autoevals before publishing this, so that we can just default the scorers to gpt-4o